admin-dev/themes/new-theme/js/components/form/form-field-toggler.ts

```diff
@@ -27,26 +27,34 @@ import {isUndefined} from '@PSTypes/typeguard';
 
 const {$} = window;
 
+// eslint-disable-next-line no-shadow
```

Comment:
can you explain why this comment is needed ? the documentation about this lint is not so clear 🤔

Reply:
Mmmh, I guess it's not needed!